### ISSUE TITLE
Only execute "run" command after onEnd is done

### ISSUE THREAD

**Issue by Utsav2** [rsms/estrella#38]
Currently, estrella runs the "run" command before "onEnd" is complete. Sometimes, onEnd can perform actions like moving directories which is useful to execute before the run command is executed. Perhaps this behavior could be configurable.

**Comment by rsms** [comment]
Oh, good idea

**Comment by rsms** [comment]
```
$ ./build.js
Wrote out/main.js (390B, 7.43ms)
user onEnd begin
user onEnd done
Hello from a running program /Users/rsms/src/estrella/examples/run/out/main.js
env["ESTRELLA_PATH"] = /Users/rsms/src/estrella/dist/estrella.js
env["ESTRELLA_VERSION"] = 1.4.0
Hello...
Hello on stderr
...world!
```

https://github.com/rsms/estrella/blob/33ede3c7bad132dfbe6901eb34bccb50694b9584/examples/run/build.js#L10-L17

**Comment by rsms** [comment]
Shipped in [v1.4.1](https://github.com/rsms/estrella/releases/tag/v1.4.1)

**Comment by Utsav2** [comment]
Thank you!
